CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Changelog
 
+### v1.4.13
+#### Fixed
+- Fixed failing MonthlyDataTab tests.
+
+### v1.4.12
+#### Updated
```

Comment:
Since `1.4.12` wasn't published, why not bundle both changes together into `1.4.12` instead of `1.4.13`?